ts/src/apex.ts

```diff
@@ -0,0 +1,1940 @@
+        //     "indexPrice": "40828.94",
+        //     "openInterest": "2036854775808",
+        //     "turnover24h": "5626085.23749999",
+        //     "volume24h": "169.317",
+        //     "fundingRate": "0",
+        //     "predictedFundingRate": "0",
+        //     "nextFundingTime": "10:00:00",
+        //     "tradeCount": 100
+        // }
+        //
+        const timestamp = this.milliseconds ();
+        const marketId = this.safeString (ticker, 'symbol');
+        market = this.safeMarket (marketId, market, undefined);
+        const symbol = this.safeSymbol (marketId, market, undefined);
+        const last = this.safeString (ticker, 'lastPrice');
+        const percentage = this.safeString (ticker, 'price24hPcnt');
+        const percent = Precise.stringMul (percentage, '100');
+        const open = Precise.stringDiv (last, Precise.stringMul ('1', percentage), 8);
+        const quoteVolume = this.safeString (ticker, 'turnover24h');
+        const baseVolume = this.safeString (ticker, 'volume24h');
+        const high = this.safeString (ticker, 'highPrice24h');
+        const low = this.safeString (ticker, 'lowPrice24h');
+        return this.safeTicker ({
+            'symbol': symbol,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'high': high,
+            'low': low,
+            'bid': undefined,
+            'bidVolume': undefined,
+            'ask': undefined,
+            'askVolume': undefined,
+            'vwap': undefined,
+            'open': open,
+            'close': last,
+            'last': last,
+            'previousClose': undefined,
+            'change': undefined,
+            'percentage': percent,
+            'average': undefined,
+            'baseVolume': baseVolume,
+            'quoteVolume': quoteVolume,
+            'markPrice': this.safeString (ticker, 'markPrice'),
+            'indexPrice': this.safeString (ticker, 'indexPrice'),
+            'info': ticker,
+        }, market);
+    }
+
+    /**
+     * @method
+     * @name apex#fetchTicker
+     * @description fetches a price ticker, a statistical calculation with the information calculated over the past 24 hours for a specific market
+     * @see https://api-docs.pro.apex.exchange/#publicapi-v3-for-omni-get-ticker-data-v3
+     * @param {string} symbol unified symbol of the market to fetch the ticker for
+     * @param {object} [params] extra parameters specific to the exchange API endpoint
+     * @returns {object} a [ticker structure]{@link https://docs.ccxt.com/#/?id=ticker-structure}
+     */
+    async fetchTicker (symbol: string, params = {}): Promise<Ticker> {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchTicker() requires a symbol argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request: Dict = {
+            'symbol': market['symbol'],
```

Comment:
we shouldn't use `market['symbol']` in the request, because that represents the ccxt-unified symbol, it should be `market['id']`